lib/mix/tasks/release.ex

```diff
@@ -354,6 +354,7 @@ defmodule Mix.Tasks.Release do
     # Re-package release with modifications
     file_list = File.ls!(rel_dest_path(name))
       |> Enum.reject(&(&1 == erts))
+      |> Enum.reject(&(&1 == "tmp"))
```

Comment:
Rather than iterating twice, you can combine the two `reject` calls and just do:

```elixir
|> Enum.reject(fn n -> n in [erts, "tmp"] end)
```

Could you make that change?